client/src/modules/dashboard/Chapters/components/ChapterForm.tsx

```diff
@@ -16,20 +17,65 @@ interface ChapterFormProps {
   submitText: string;
 }
 
-type Fields =
-  | [keyof ChapterFormData, boolean, boolean]
-  | [keyof ChapterFormData, boolean]
-  | [keyof ChapterFormData];
+type Fields = {
+  key: keyof ChapterFormData;
+  placeholder: string;
+  label: string;
+  required: boolean;
+  type: string;
+};
 
 const fields: Fields[] = [
-  ['name', true],
-  ['description', true],
-  ['details', true],
-  ['city', true],
-  ['region', true],
-  ['country', true],
-  ['category', true],
-  ['imageUrl', true],
+  {
+    key: 'name',
+    label: 'Chapter name',
+    placeholder: 'freeCodeCamp',
+    required: true,
+    type: 'text',
+  },
+  {
+    key: 'description',
+    label: 'Description',
+    placeholder:
+      'freeCodeCamp is a non-profit organization that helps people learn to code for free',
```

Comment:
```suggestion
      'freeCodeCamp is a nonprofit organization that helps people learn to code for free',
```
Consistency with 'category' below.